linkedList.js

```diff
@@ -3,6 +3,83 @@
  * @description  Main Module
  * @return {Object} an object exposing methods to be used to manipulate a linked list
  */
-function linkedListGenerator(){
+function linkedListGenerator() {
+  //when it says it should have a "method", it means function
+  let head = null;
+  let tail = null;
 
-}
\ No newline at end of file
+  let getHead = function() {
+    return head;
+  };
+
+  let getTail = function() {
+    return tail;
+  };
+
+  let add = function(value) {
+    let newNode = {
+      value: value,
+      next: null
+    };
+
+    if (head == null) {
+      head = newNode;
+    }
+
+    if (tail !== null) {
+      tail.next = newNode;
+    }
+    tail = newNode;
+    return newNode;
+  };
+
+  let get = function(index) {
+    // console.log(head);
+    let indexCount = 0; //starting index
+    let currentNode = head; //starting point
+    while (indexCount < index && currentNode !== null) {
+      currentNode = currentNode.next;
+      indexCount++;
+    }
+    if (currentNode == null) {
```

Comment:
You don't need this if statement since you're handing this case in the while loop